src/quo2/screens/user_avatar.cljs

```diff
@@ -0,0 +1,61 @@
+(ns quo2.screens.user-avatar
+  (:require [quo.react-native :as rn]
+            [quo.design-system.colors :as colors]
+            [quo.previews.preview :as preview]
+            [quo2.components.user-avatar :as quo2]
+            [reagent.core :as reagent]))
+
+(def descriptor [{:label   "Size:"
+                  :key     :size
+                  :type    :select
+                  :options [{:key   :big
+                             :value "Big"}
+                            {:key   :medium
+                             :value "Medium"}
+                            {:key   :small
+                             :value "Small"}
+                            {:key   :xs
+                             :value "x Small"}
+                            {:key   :xxs
+                             :value "xx Small"}
+                            {:key   :xxxs
+                             :value "xxx Small"}]}
+                 {:label "Online status"
+                  :key   :online?
+                  :type  :boolean}
+                 {:label "Status Indicator"
+                  :key   :status-indicator?
+                  :type  :boolean}
+                 {:label "Identicon Ring"
+                  :key   :ring?
+                  :type  :boolean}
+                 {:label "Full name separated by space"
+                  :key   :full-name
+                  :type  :text}
+                 {:label "Profile Picture URL"
+                  :key   :profile-picture
+                  :type  :text}])
+
+(defn cool-preview []
+  (let [state  (reagent/atom {:full-name "empty name"
```

Comment:
consider changing this so the default initials are A Y and comparing the designs is faster.

Reply:
Got a comment from Andre on making it as empty name